lizard_auth_server/models.py

```diff
@@ -543,7 +588,9 @@ def activate(self, data):
 
             # create and fill the profile
             # this sets the additional attributes on the User model as well
-            profile = user.get_profile()
+            # get_profile deprecated in Django >= 1.7
+            # profile = user.get_profile()
```

Comment:
Dit soort commentaar is op dit moment erg handig!

Wel moeten we dan over een maandje de oude commentaren een keer opschonen, maar dat komt dan wel.